org.kde.WaylandDecoration.QAdwaitaDecorations.appdata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.kde.WaylandDecoration.QAdwaitaDecorations</id>
```

Comment:
this should be probably be shipped with QAdwaitaDecorations

also I wonder if we shouldn't use a org.fedora.WaylandDecoration.QAdwaitaDecorations instead of the org.kde prefix

Reply:
I believe it has to use the `org.kde.WaylandDecoration` prefix as it's an extension.